src/pages/profiles/[userId].tsx

```diff
@@ -62,7 +65,8 @@ export default function Page() {
   const [profile, setProfile] = useState<UserProfile>();
   useEffect(() => setProfile(old?.profile), [old]);
 
-  const updateProfile = (k: keyof UserProfile, v: string) => {
+  const updateProfile = (k: keyof UserProfile, v: string | 
+    { x: number; y: number; zoom: number }) => {
```

Comment:
in UserProfile.ts, define a type `ImageParams` or something like that and use it here.